docsbox/Dockerfile

```diff
@@ -52,7 +52,7 @@ RUN curl --remote-name https://exiftool.org/Image-ExifTool-$EXIF_TOOL_VERSION.ta
 
 # Install dcs
 ADD requirements.txt /home/docsbox/requirements.txt
-RUN pip3 install -r /home/docsbox/requirements.txt
+RUN pip3 install --break-system-packages -r /home/docsbox/requirements.txt
```

Comment:
Needed to put this --break-system-packages because of the weird relation of requirements in this project. Because we are using both pip3 install AND apt-get for getting python packages this needs to be here.
In the future I would suggest to put everything in one or the other (probably all on apt-get because uno package can only be obtained from there...)